Updated redemption through the direct link (#76235)
Azure/azure-content: articles/active-directory/external-identities/redemption-experience.md

 - Sometimes the invited user object may not have an email address because of a conflict with a contact object (for example, an Outlook contact object). In this case, the user must click the redemption URL in the invitation email.
 - The user may sign in with an alias of the email address that was invited. (An alias is an additional email address associated with an email account.) In this case, the user must click the redemption URL in the invitation email.

> Sometimes the invited external guest user's email may conflict with an existing [Contact object](https://docs.microsoft.com/en-us/graph/api/resources/contact?view=graph-rest-1.0&preserve-view=true), resulting in the guest user being created without a proxyAddress. This is a known limitation that will prevent guest users signing in or redeeming invitation through the direct link using [SAML/WS-Fed IdP](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/direct-federation), [Microsoft Accounts](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/microsoft-account), [Google Federation](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/google-federation), and [Email One-Time Passcode](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/one-time-passcode) accounts.

Reviewer: ```suggestion
Sometimes the invited external guest user's email may conflict with an existing [Contact object](https://docs.microsoft.com/en-us/graph/api/resources/contact?view=graph-rest-1.0&preserve-view=true), resulting in the guest user being created without a proxyAddress. This is a known limitation that prevents guest users from signing in or redeeming an invitation through a direct link using [SAML/WS-Fed IdP](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/direct-federation), [Microsoft Accounts](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/microsoft-account), [Google Federation](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/google-federation), or [Email One-Time Passcode](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/one-time-passcode) accounts.
```

> When this occurs, follow these steps to unblock users who aren't able to redeem due to a conflicting [Contact object](https://docs.microsoft.com/en-us/graph/api/resources/contact?view=graph-rest-1.0&preserve-view=true):

Reviewer: ```suggestion
To unblock users who can't redeem an invitation due to a conflicting [Contact object](https://docs.microsoft.com/en-us/graph/api/resources/contact?view=graph-rest-1.0&preserve-view=true), follow these steps:
```

> 1. Delete the conflicting Contact object

Reviewer: ```suggestion
1. Delete the conflicting Contact object.
```

> 2. Delete the guest user in the Azure portal (that should be in pending state under "Invitation accepted")

Reviewer: ```suggestion
2. Delete the guest user in the Azure portal (the user's "Invitation accepted" property should be in a pending state).
```

> 3. Re-invite the guest user

Reviewer: ```suggestion
3. Re-invite the guest user.
```
